TACC/Core-CMS
# task/FP-1780 -- core cms logging

## Overview

Add logging to Core CMS

## Related

- [FP-1780](https://jira.tacc.utexas.edu/browse/FP-1780)

## Changes

- Logging to /var/log/cms within the CMS container

## Testing

1. Set your docker-compose.dev.yml core-cms image to `taccwma/apcd-cms:32916dc`
2. Start your local development environment
3. Get a shell on the cms container and see that logs appear in `/var/log/cms/cms.log`

## UI

…
